x/auth/keeper/keeper.go

```diff
@@ -18,18 +19,18 @@ import (
 // encoding/decoding library.
 type AccountKeeper struct {
 	key           sdk.StoreKey
-	cdc           types.Codec
+	Cdc           codec.Marshaler
```

Comment:
why is this public? use a getter instead if you need to use it outside this package.

Reply:
Sure, will replace with getter